pkgs/by-name/ri/rime-japanese/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "rime-japanese";
+  version = "unstable-20230802";
```

```suggestion
  version = "0-unstable-2023-08-02";
```

this is the recommended version format for packages that don't have tags.